src/components/sections/hero/hero.module.css

```diff
@@ -6,8 +6,7 @@
 }
 
 .secondaryColor {
-  background-image: url("/headerAnimation.svg");
-  background-size: cover;
+  background-color: var(--secondary-green3);
```

Comment:
you could also remove it all together I believe.

Reply:
just wanted to preserve the secondaryColor distinction for landing pages